app/src/main/res/layout-land/ongoing_topic_list_fragment.xml

```diff
@@ -57,6 +57,7 @@
         android:paddingTop="28dp"
         android:paddingBottom="88dp"
         android:scrollbars="none"
+        android:tag="TAG_ONGOING_RECYCLER_VIEW"
```

Comment:
Set this in `strings.xml` and use it whereever you want. If in future someone changes the tag here, even the test cases will start failing, so `strings.xml` approach will work in a better way.

Reply:
done